File: 58-length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {

        int n = s.length()-1;

        while(n>=0){
            if(s.charAt(n)!=' ')break;
            --n;
        }
        int count=0;
        while(n>=0){
            if(s.charAt(n)==' ')break;
            --n;
            count++;
        }
        return count;
        
    }
}

File: 58-length-of-last-word/length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
/*
    s=s.trim();
    s=s.split(" ");
    return s[s.length-1].length;
    */

    let n = s.length-1;
    while(n>=0){
        if(s[n]===" "){
            --n;
        }else{
            break;
        }
    }

    let count=0;
    while(n>=0){
        if(s[n]!=" "){
            --n;
            ++count;
        }else{
            break;
        }
    }
    return count;
    
};